src/language.rs

```diff
@@ -46,11 +58,15 @@ macro_rules! define_languages {
             }
 
             pub fn get_colors(&self) -> Vec<Color> {
-                match *self {
-                    $(
-                        Language::$name => $colors,
-                    )*
-                    Language::Unknown => vec![Color::White],
+                let colors = match *self {
+                    $( Language::$name => define_colors ! ( $($colors)* ), )*
+                    Language::Unknown => define_colors ! ( vec![Color::White] ),
+                };
+                let use_true_color = env::var("COLORTERM");
+                if use_true_color.is_ok() && use_true_color.unwrap().to_lowercase() == "truecolor" && colors.true_colors.is_some() {
+                    colors.true_colors.unwrap()
+                } else {
+                    colors.basic_colors
                 }
```

Comment:
See if this will work. Checking `is_ok` and then unwrapping should probably be replaced with pattern matching, with either `if let` or `match`.
```suggestion
                if let (Ok("truecolor"), Some(colors)) = (use_true_color.map(|s| s.to_lowercase()), colors.true_colors) {
                    colors
                } else {
                    colors.basic_colors
                }
```